rdkit/Chem/UnitTestCatalog.py

```diff
@@ -14,7 +14,7 @@
 from rdkit import RDConfig
 from rdkit import Chem
 from rdkit.Chem import FragmentCatalog, BuildFragmentCatalog
-from rdkit.six.moves import cPickle
+from rdkit.six.moves import cPickle  # @UnresolvedImport
```

Comment:
Sounds reasonable, but I do want to re-point out that the only value most of these add are to make it so that developers who work the same way you do aren't bothered by the tools they have chosen to use.  :-)